plugins/multiserver/com_osvr_Multiserver_Sensics_zSight.json

```diff
@@ -0,0 +1,18 @@
+{
+  "deviceVendor": "Sensics",
+  "deviceName": "Sensics zSight HMD",
+  "author": "Virendra Singh",
+  "version": 1,
+  "lastModified": "2015-09-28",
+  "interfaces": {
+    "tracker": {
+      "count": 1,
+      "bounded": true,
+      "position": false,
+      "orientation": true
+    }
+  },
+  "semantic": {
+    "untared_orientation": "tracker/0"
```

Comment:
It has been working for me without any problems till now. I just had to add display config in osvr_server_config.